tests/test_toa_writer.py

```diff
@@ -45,34 +43,54 @@ def test_roundtrip_topo_toa_Tempo2format(self):
         ts = toa.get_TOAs_list([t1], ephem="DE421")
         ts.write_TOA_file("testtopo.tim", format="Tempo2")
         ts2 = toa.get_TOAs("testtopo.tim")
-        print(ts.table, ts2.table)
         assert np.abs(ts.table["mjd"][0] - ts2.table["mjd"][0]) < 1.0e-15 * u.d
         assert np.abs(ts.table["tdb"][0] - ts2.table["tdb"][0]) < 1.0e-15 * u.d
 
+    def test_commenting_toas(self):
+        # Create a barycentric TOA
+        t1time = Time(58534.0, 0.0928602471130208, format="mjd", scale="utc")
+        t1 = toa.TOA(t1time, obs="gbt", freq=0.0)
+        t2 = toa.TOA(t1time, obs="gbt", freq=0.0)
+        ts = toa.get_TOAs_list([t1, t2], ephem="DE421")
+        assert ts.ntoas == 2
+        ts.write_TOA_file("testtopo.tim", format="Tempo2")
+        ts2 = toa.get_TOAs("testtopo.tim")
+        assert ts2.ntoas == 2  # none should be commented
+        ts.table[0]["flags"]["cut"] = "do not like"  # default comment flag
```

Comment:
What happens when we use flags with spaces in them? Does PINT handle this correctly? (What would correct handling look like - there isn't a defined form of quoting is there?)

Reply:
Yeah, I don't even know what that should look like.  Especially if a flag (or the flag's value) with spaces is in a .tim file.

Reply:
I changed the value to have s/ /_/ in this case.